unlock-app/src/components/creator/lock/KeyList.js

```diff
@@ -100,7 +100,7 @@ const Row = styled.div`
   margin-left: 48px;
   width: 100%;
   display: grid;
-  grid-template-columns: 2fr 3fr 3fr;
+  grid-template-columns: 3fr 1fr 4fr;
```

Comment:
I think we should probably use a fixed size field for the first column since this will always be ethereum addresses of the same size?

Reply:
And same for the 2nd one which is a date field?